src/nidigital/metadata/functions.py

```diff
@@ -39,6 +39,8 @@
             },
             {
                 'direction': 'in',
+                'is_repeated_capability': True,
+                'repeated_capability_type': 'sites',
```

Comment:
Please make the metadata changes in internal metadata files.

Reply:
Done